cegui/src/text/LegacyTextParser.cpp

```diff
@@ -216,6 +220,21 @@ bool LegacyTextParser::parse(const String& inText, std::u32string& outText,
     return true;
 }
 
+Sizef getRespectRatioSize(Sizef reqSize, const Sizef& orgSize) {
+	float orgRatio = orgSize.d_width / orgSize.d_height;
+	float reqRatio = reqSize.d_width / reqSize.d_height;
```

Comment:
**Minor**: these two can be declared `const`. Not for astonishing performance boost but for clear statement that they are not going to change. This improves readability a bit.

Reply:
fixed